docs/integrations/sources/linkedin-ads-migrations.md

```diff
@@ -0,0 +1,29 @@
+# LinkedIn Ads Migration Guide
+
+## Upgrading to 1.0.0
+
+Version 1.0.0 introduces changes in primary key for all *-analytics streams (including custom ones).
```

Comment:
please add the stream names here as you did in the metadata file

Reply:
updated